content/posts/2023/reana-0.9.0.md

```diff
@@ -0,0 +1,161 @@
+
+If your workflow needs to use data that is managed by a Rucio system, you can
+follow the documentation that describes the [Rucio authentication method](https://docs.reana.io/advanced-usage/access-control/rucio/)
+to learn more about it.
+
+### Kerberos authentication for workflow orchestration
+
+Previously, you had to set up Kerberos authentication for each step of your
+workflow, one at a time. You can now easily enable Kerberos authentication
+for your entire workflow by adding just one line in the specification file. This
+makes it much more convenient and efficient. This technique is especially useful
+for workflows that need to access files in restricted storage in multiple steps.
+
+Please check our documentation about [enabling Kerberos support for an entire workflow](https://docs.reana.io/advanced-usage/access-control/kerberos/#setting-kerberos-requirement-for-whole-workflow)
+to learn more about this feature.
+
+### Notifications to inform when critical levels of quota usage is reached
+
+A new notifications feature has been added to REANA web interface to alert users
+when their [quota](https://docs.reana.io/advanced-usage/user-quotas) is
+approaching or has reached its limit. This feature is intended to help users
+stay within their allocated Disk space and CPU consumption limits and inform
+them when deletion of unnecessary workflow runs should be performed in order to
+liberate some resources.
+
+![Quota notifications](/images/reana-0.9.0-notifications.png)
+
+## What's new for the administrators?
+
+### Configure global workspace retention rules
+
+Workspace retention rules are giving users the possibility to remove unwanted files from their workflows' workspaces.
+However, administrators can also define a global workspace file retention period, after which all the files that are not listed as inputs or outputs of a workflow will be deleted, regardless of user-defined workspace retention rules.
+This can be achieved by setting the `workspaces.retention_rules.maximum_period` Helm value to the desired amount of retention days.
+Please note that, by default, files are retained forever.
+
+For more information about this, see the related documentation page about [configuring global workspace retention rules](https://docs.reana.io/administration/configuration/configuring-global-workspace-retention-rules).
+
+### Configure an additional volume for infrastructure components
+
+Up until now, REANA has used only one shared storage volume to store all the workflows' workspaces, together with the database's and message broker's data.
+This means that, if the shared storage became full, some of the REANA components would fail due to the lack of free disk space.
+While this is still the default behaviour of REANA, now it is also possible to configure a second infrastructure storage volume that can be used to separate workflows' workspaces from the data stored by REANA's infrastructure components.
```

Comment:
separate workflows's workspaces from... -> 

I would inverse the order, to better highlight we are "saving" infra volume.  So say something like

...to better separate the data stored by the REANA infrastructure components from the runtime data generated by user workflows.